apps/rush-lib/src/api/PackageManagerFeatureSet.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as semver from 'semver';
+import { RushConstants } from '../logic/RushConstants';
+
+/**
+ * This represents the available Package Manager tools as a string
+ * @public
+ */
+export type PackageManager = 'pnpm' | 'npm' | 'yarn';
```

Comment:
There isn't already an enum for this?

Reply:
This is moved from a different source file, but otherwise unchanged.